src/mitigate_branch_outlet_backpool.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env python3
+
+import argparse
+import sys
+from collections import OrderedDict
+from os import remove
+from os.path import isfile
+
+import geopandas as gpd
+import numpy as np
+import pandas as pd
+import rasterio
+from shapely import ops, wkt
+from shapely.geometry import LineString, Point
+from shapely.ops import split as shapely_ops_split
+from tqdm import tqdm
+from collections import Counter
+
+
+import build_stream_traversal
+from utils.fim_enums import FIM_exit_codes
+from utils.shared_functions import getDriver, mem_profile
+from utils.shared_variables import FIM_ID
+
+@mem_profile
```

Comment:
@EmilyDeardorff This just caught my eye. The `@mem_profile` decorator function has been removed in the latest dev branch. You would probably catch this after merging the latest dev, but figured I'd give you a heads up.

Reply:
Fixed this! Good catch,